Gemfile

```diff
@@ -4,6 +4,7 @@ source 'https://rubygems.org'
 
 git_source(:github) { |repo_name| "https://github.com/#{repo_name}" }
 
+gem 'pry'
```

Comment:
hooray! I always just add this manually when I'm debugging.

Reply:
You can add gems if you want 😅

Reply:
I know, just never got around to it : )